### ISSUE TITLE
Freeze version of installed python modules in Docker file

### ISSUE THREAD

**Issue by ntraut** [bids-apps/baracus#13]
If one rebuild the docker image using the Dockerfile, it installs a version of pybids which is too new for this app. I would suggest to freeze the version of pip installed modules, like:

```
RUN pip install nibabel==2.2.0
RUN pip install pybids==0.3 grabbit==0.0.8
RUN pip install duecredit==0.6.3
```

**Comment by fliem** [comment]
Thanks for reporting this. I'll fix this after #14

**Comment by fliem** [comment]
closed by #15
